README.md

```diff
@@ -14,18 +9,48 @@ This repo uses [Git Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodu
 
 To clone using HTTPS:
 ```
-git clone https://github.com/FreeRTOS/Lab-Project-coreMQTT-Agent.git --recurse-submodules
+git clone https://github.com/FreeRTOS/coreMQTT-Agent-Demos.git --recurse-submodules
 ```
 Using SSH:
 ```
-git clone git@github.com:FreeRTOS/Lab-Project-coreMQTT-Agent.git --recurse-submodules
+git clone git@github.com:FreeRTOS/coreMQTT-Agent-Demos.git --recurse-submodules
 ```
 
 If you have downloaded the repo without using the `--recurse-submodules` argument, you need to run:
 ```
 git submodule update --init --recursive
 ```
 
+## Getting started
+The [documentation page](https://freertos.org/mqtt/mqtt-agent-demo.html) for this repository contains information on the MQTT agent and the contained demo project.  There is also a [supplemental documentation page](https://freertos.org/ota/ota-mqtt-agent-demo.html) that describes how to run an Over the Air (OTA) update agent as one of the RTOS tasks that share the same MQTT connection.
```

Comment:
OTA goes as **Over-the-Air** in the branding: https://docs.aws.amazon.com/freertos/latest/userguide/freertos-ota-dev.html
```suggestion
The [documentation page](https://freertos.org/mqtt/mqtt-agent-demo.html) for this repository contains information on the MQTT agent and the contained demo project.  There is also a [supplemental documentation page](https://freertos.org/ota/ota-mqtt-agent-demo.html) that describes how to run an Over-the-Air (OTA) update agent as one of the RTOS tasks that share the same MQTT connection.
```